paddle/fluid/operators/conv_cudnn_op.cu.cc

```diff
@@ -20,10 +20,10 @@ limitations under the License. */
 #include "paddle/fluid/platform/cudnn_helper.h"
 #include "paddle/fluid/platform/float16.h"
 
-DEFINE_bool(cudnn_deterministic, true,
+DEFINE_bool(cudnn_deterministic, false,
             "Whether allow using an autotuning algorithm for convolution "
             "operator. The autotuning algorithm may be non-deterministic. If "
-            "false, the algorithm is deterministic.");
+            "true, the algorithm is deterministic.");
```

Comment:
`cudnn_deterministic` is used in `pool_cudnn_op.cu.cc` too.

Reply:
Yes. This flag does mean `deterministic` in `pool_cudnn_op.cu.cc`.